extensions/wikia/FandomCreator/Hooks.php

```diff
@@ -83,11 +73,74 @@ public static function onDesignSystemApiGetAllElements( WikiaDispatchableObject
 			];
 		}
 
-		if ( !empty( $theme->graphics->header ) ) {
-			$currentData['community-header']['background_image'] = $theme->graphics->header;
+		if ( !empty( $community->theme->graphics->header ) ) {
+			$data['community-header']['background_image'] = $community->theme->graphics->header;
+		}
+
+		// remove "explore" menu since it's forcefully added by the DS api. array_values is needed because array_filter
+		// turns the array into an assoc. array, so when json_encoded navigation is an object instead of an array
+		$data['community-header']['navigation'] = array_values(array_filter($data['community-header']['navigation'], function($navItem) {
+			$type = isset($navItem['type']) ? $navItem['type'] : false;
+			$titleType = isset($navItem['title']['type']) ? $navItem['title']['type'] : false;
+			$titleKey = isset($navItem['title']['key']) ? $navItem['title']['key'] : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)